Backoffice/LeftPanel/Strategies/ErrorPagesPanelStrategy.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\LeftPanel\Strategies;
+
+use OpenOrchestra\ModelInterface\Model\ReadNodeInterface;
+
+/**
+ * Class ErrorPagesPanelStrategy
+ */
+class ErrorPagesPanelStrategy extends AbstractLeftPanelStrategy
+{
+    const ROLE_ACCESS_TREE_NODE = 'ROLE_ACCESS_TREE_NODE';
```

Comment:
It is also defined in TreeNodePanelStrategy, but I'm not fan of one strategy refering to another. Alternativly we could define that const in the AbastractStrategy if you're ok.